tools/azure-plugin/generator/Azure.Generator/src/AzureClientPlugin.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp;
+using Microsoft.Generator.CSharp.ClientModel;
+using System.ComponentModel.Composition;
+
+namespace Azure.Generator;
+
+/// <summary>
```

Comment:
This doesn't get merged until these comments get fixed.

Reply:
added comments